### PR TITLE
Add evidence: koute - promote to rank III, May 2024

### PR DESCRIPTION
**Pull request by koute**
[Rendered](https://github.com/polkadot-fellows/Evidences/blob/a8fc017ffdd53a8c1665c46a5bed07999c3ff6ae/evidence/koute/0001-koute-retain-II-2024.md)

### CODE REVIEW

**Review by ggwpez — APPROVED** [review]
Maybe we should amend the Fellowship Manifesto to include PolkaVM in its mandate.

**Review by bkchr — APPROVED** [review]

File: evidence/koute/0001-koute-evidence-2024-05-09.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+| ---------------- | ---------------------|
+| **Report Date**  | 2024/05/09           |
+| **Submitted by** | Jan Bujak            |
+
+## Member details
+
+- Matrix username: @jan:parity.io
+- Polkadot address: 15tRXfXoZXkjScB3Awv8s2saPjaicKYAyL1WZ3JP5kycoG9n
+- Current rank: 2
+- Date of initial induction: 2024/01/08
```
Comment (anaelleltd) [inline evidence/koute/0001-koute-evidence-2024-05-09.md:13]: Jan was part of the Technical Fellowship [seeding](https://github.com/polkadot-fellows/seeding).
So, the Date of initial induction should read "seeding".

Comment (koute) [reply evidence/koute/0001-koute-evidence-2024-05-09.md]: Done; thanks.

**Review by gilescope — APPROVED** [review]
LGTM

**Pull request merged**